### DISCUSSION TITLE
Restoring/converting NVM gen500 backup to ZAC93 800 series results in serialisation error

### DISCUSSION THREAD

**Discussion by cadwizzard** [zwave-js/zwave-js#7214] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
NVM restore fails

What did you expect to happen instead?
NVM restore to complete without issue.

(I have tried this process with Home assistant and Zwave JSUI, and  Ubunto and the nvmedit tool directly, with same results
In HA, Add ZAC93 GPIO module (with latest firmware 1.5)
  1. Make a backup of NVM Aeotec Gen500 controller
  2. Change the Zwave serial port settings to AMA0 for the GPIO 800 controller and press save
  3. Restore the 500 backup to the Zooz 800 controller
  4. conversion completes 100% but the restore fails with error.
![Screenshot 2024-09-30 185443](https://github.com/user-attachments/assets/c5a0d690-bb6a-4c38-821a-8170eeae12ef)

Anything else we should know?
I have been able to convert and restore the gen500 backup also to an (Aeotec zstick7) 700 series controller on both jsui and nvmedit with no issue, but trying to then restore via jsui or convert seperately using nvmedit, also results in the same error:
![Screenshot 2024-09-30 213000](https://github.com/user-attachments/assets/fa71b89b-e288-428b-b65b-d724ed5f4ed4)

I have searched for this error, but cannot find anyone else with any reference to it.

The debug log contains an attempted restore from firstly a Gen500+ Aeotec controller and then straight after an Aeotec Zstick7,  both resulting is same serialisation error

### Software versions

nvmedit 1.34.0

Z-Wave JS UI: 3.130

Home Assistant Z-Wave Integration: 
Driver Version:
13.4.0
Server Version:
1.38.0

### Z-Wave Controller (Stick/Dongle/...)

Aeotec Gen500+ : fw1.2
Zooz ZAC93 : fw1.50 SDK 7.22.1

### Device information

Manufacturer: Zooz
Model name: ZAC93
Node ID: 1

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-10-01.log](https://github.com/user-attachments/files/17212694/zwavejs_2024-10-01.log)

**Comment by AlCalzone** [comment]
Duplicate of https://github.com/zwave-js/node-zwave-js/discussions/7199
